sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/query.sql

```diff
@@ -69,6 +69,16 @@ SELECT
   COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS dau,
   COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS wau,
   COUNT(DISTINCT client_id) AS mau,
+  COUNT(
```

Comment:
Using a DISTINCT here for unique client count, which slightly differs from the [proposed query](https://sql.telemetry.mozilla.org/queries/89768/source).